features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/AddOperation.jsx

```diff
@@ -290,7 +290,7 @@ function AddOperation(props) {
                         name='target'
                         value={newOperations.target}
                         onChange={({ target: { name, value } }) => newOperationsDispatcher(
-                            { type: name, value: value.startsWith('/') ? value : `/${value}` },
+                            { type: name, value },
```

Comment:
This string comprehension append a `/` to the path if the user forgot to add it, I'm not sure how removing that going to fix `default` path issue

Reply:
When it comes to webhooks adding a '/' doesn't make sense for topics. However the original issue that is mentioned is eventhough the default topic name is _default in UI in DB it is added as /_default and that causes the runtime issue. Therefore as a fix we can remove adding / for topics